### ISSUE TITLE
[use mirror from]: issue when DE is not in english

### ISSUE THREAD

**Issue by scachemaille** [manjaro/pamac#155]
if the user use a different language display than english (tested with french) when the user change the **_Use mirrors from_** to "Worldwide" (in english as in french it's translated to Monde entier). 
pamac don't comment the **_OnlyCountry=_** entry in **_pacman-mirrors.conf_** but put the translation of worldwide as country in the option. 

pamac V4.1.1-1

**Comment by papajoker** [comment]
[line 126](https://github.com/manjaro/pamac/blob/master/src/mirrors_config.vala#L126)
test if value is not in list of country is not more robust ?

**Comment by guinux** [comment]
I can't reproduce it, for me it works as expected even in French. Does someone else can ?

**Comment by scachemaille** [comment]
A problem was reported in French forum and u tried to understand it and I could reproduce it.  It's why I reported it.  
My test was on plasma 5.7 I will check the DE if the original poster in the French forum

**Comment by scachemaille** [comment]
hmmm.  it seems it's related to the fact that the root display language or system language) is not the same as my KDE. I tried to change the display language to FR in manjaro-settings-manager and it seems to work.
Yes. now if I set the preffered language to English in KDE settings (still with French in manjaro-settings-manager as display language) pamac same OnlyCountry=Worlwide and don't comment it
